src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/ExportToolbarAction.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import {action, observable} from 'mobx';
+import React from 'react';
+import {translate} from '../../../utils/Translator';
+import Overlay from '../../../components/Overlay';
+import Form from '../../../components/Form';
+import SingleSelect from '../../../components/SingleSelect';
+import resourceEndpointRegistry from '../../../services/ResourceRequester/registries/ResourceRouteRegistry';
+import {buildQueryString} from '../../../utils/Request';
+import exportToolbarActionStyles from './exportToolbarAction.scss';
+import AbstractToolbarAction from './AbstractToolbarAction';
+
+export default class ExportToolbarAction extends AbstractToolbarAction {
+    @observable showOverlay = false;
+    @observable delimiter: string = ';';
+    @observable enclosure: string = '"';
+    @observable escape: string = '\\';
+    @observable newLine: string = '\\n';
+
+    getNode() {
+        return (
+            <Overlay
+                confirmDisabled={false}
+                confirmLoading={false}
+                confirmText={translate('sulu_admin.export')}
+                key="sulu_admin.export"
+                onClose={this.handleClose}
+                onConfirm={this.handleConfirm}
+                open={this.showOverlay}
+                size="small"
+                title={translate('sulu_admin.export_overlay_title')}
+            >
+                <div className={exportToolbarActionStyles.overlay}>
+                    <Form>
+                        <Form.Section colSpan={6}>
+                            <Form.Field
+                                description={translate('sulu_admin.delimiter_description')}
+                                label={translate('sulu_admin.delimiter')}
+                            >
+                                <SingleSelect onChange={this.handleDelimiterChanged} value={this.delimiter}>
+                                    <SingleSelect.Option value=";">;</SingleSelect.Option>
+                                    <SingleSelect.Option value=",">,</SingleSelect.Option>
+                                    <SingleSelect.Option value={'\\t'}>\t</SingleSelect.Option>
+                                </SingleSelect>
+                            </Form.Field>
+                            <Form.Field
+                                description={translate('sulu_admin.enclosure_description')}
+                                label={translate('sulu_admin.enclosure')}
+                            >
+                                <SingleSelect onChange={this.handleEnclosureChanged} value={this.enclosure}>
+                                    <SingleSelect.Option value={'"'}>&quot;</SingleSelect.Option>
+                                    <SingleSelect.Option value=" ">&nbsp;</SingleSelect.Option>
```

Comment:
That behaves weird, doesn't it? It doesn't look like this option can be selected... Maybe we should write "nothing" or something similar?